### ISSUE TITLE
C64_TapeBuddy64 NTSC Timing

### ISSUE THREAD

**Issue by Kalidomra** [wagiminator/C64-Collection#6]
Add NTSC timing to read and write. Seems to default to PAL timing.

**Comment by wagiminator** [comment]
Yes, you could insert a calculation formula. But I suspect that it also works without it, since the differences are comparatively minimal.
